### DISCUSSION TITLE
Test not working.

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [filamentphp/filament#3267] [General]
```php
it('can render page', function () {
    $this->get(PostResource::getUrl('create'))->assertSuccessful();
});
```

This test is not working after hitting the endpoint I got a 403 error.

Where to set up this 

```php
protected function setUp(): void
{
    parent::setUp();
 
    $this->actingAs(User::factory()->create());
}
```

**Comment by pxlrbt** [comment]
Probably a duplicate of https://github.com/filamentphp/filament/discussions/3007.

Did you set up your user model as stated in the docs?
https://filamentphp.com/docs/2.x/admin/users#authorizing-access-to-the-admin-panel
